data/deprecated.json

```diff
@@ -410,27 +410,27 @@
     },
     {
       "old": {"golf": "faiway"},
```

Comment:
is this a type for fairway?

Reply:
Yep. The `deprecated.json` file maps typos and alternative tags to tags iD understands.